src/modules/tests/movieCounter.test.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @jest-environment jsdom
+ */
+import { movieCounter } from '../movieCounter.js';
+
+test('movieCounter function should count and display the number of movies correctly', () => {
```

Comment:
- [ ] The test is not currently running. Please update the test to cover the soon-to-be-created movie counter function. Kindly try to cover as many edge cases as possible. One of the situations is when there are no movies displayed, the counter function should return 0. 🤓️
- [ ] Comment counter is not tested in the project. Please do the same type of tests for the comment counter too. 🤝️